spk/owncloud/src/wizard/upgrade_uifile.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
For all the upgrade & install uifile, as there is a lot of code would it make sense to extract it from the file so at `PRE_STRIP_TARGET` or some other point you concatenate them?  That would simplify the overall maintenance aftewards.

Reply:
Could you expand on this? Are you suggesting that I move the code which may be common into a separate file and import it using something like `source`? If so, not all the code is the same and I've tried to minimise what's in each file. If not, I'm not very familiar with `PRE_STRIP_TARGET` so I may have misinterpreted your meaning.

Reply:
This I believe is finally resolved in https://github.com/SynoCommunity/spksrc/pull/5912 with the implementation of wizard templates.